.github/dcp.yaml

```diff
@@ -0,0 +1,4 @@
+# This enables DCO bot for you, please take a look https://github.com/probot/dco
```

Comment:
@revit13 you have a typo in the filename (should be dco).

Reply:
Thanks! fixed it...